docs/en_us/platform_api/source/courses/courses.rst

```diff
@@ -0,0 +1,174 @@
+.. _Courses API Courses Resource:
+
+########################################
+Courses API Courses Resource
```

Comment:
Should all references to "catalog" be removed for this topic? Because the Course Catalog API consumes the Courses API, I moved this out from under the Course Catalog API documentation (the CC API docs link to it).